connect/clients/nats.py

```diff
@@ -336,29 +374,114 @@ async def create_nats_client(servers: List[str]) -> Optional[NatsClient]:
     """
     Create a NATS client for any NATS server or NATS cluster configured to accept this installation's NKey.
 
-    :param servers: List of one or more NATS servers.  If multiple servers are
-    provided, they should be in the same NATS cluster.
+    :param servers: List of one or more NATS servers in the same NATS cluster.
     :return: a connected NATS client instance
     """
     settings = get_settings()
 
-    client = NatsClient()
-    await client.connect(
+    client = await nats.connect(
+        verbose=True,
```

Comment:
We can remove verbose=True in a later update.  This does not appear to affect logging at all, in connect or nats-server, but would require retesting on all platforms to remove now.

Reply:
Looks like verbose will raise an error - https://github.com/nats-io/nats.py/blob/2708a2600de3be35103b14c66941ffd6f681ac04/nats/aio/client.py#L1667 but I didn't trace down all of the options usage.